packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository.test.ts

```diff
@@ -2354,7 +2490,7 @@ describe('SavedObjectsRepository', () => {
       });
     });
   });
-
+  // @TINA: expand test suite to include setting managed
```

Comment:
Is this TODO done? 😄

Reply:
oops, that's a left-over from splitting the work.